### PR TITLE
Updated readme, in an attempt to make it more readable

### PR DESCRIPTION
**Pull request by rdnuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,30 +35,34 @@ if the lib is actually installed.
...
+minetest.register_node("mymod:mynode", {
+	description = S("My Fabulous Node"),
+	<...>
+})
+```
 
-Then, you create a `locale' directory inside your mod directory, with files
+Then, you create a 'locale' directory inside your mod directory, with files
 named after the two-letter ISO Language Code of the languages you want to
 support. Here's an example for a Spanish locale file (`es.txt'):
```
Comment (ShadowNinja) [inline README.md]: You need consistent quotes around `es.txt`.

File: README.md [hunk 2]
```diff
@@ -34,30 +35,34 @@ if the lib is actually installed.
...
 
-	# Lines beginning with a pound sign are comments and are effectively ignored
-	# by the reader. Note that comments only span until the end of the line;
-	# there's no support for multiline comments.
-	Hello, World! = Hola, Mundo!
-	String with\nnewlines = Cadena con\nsaltos de linea
-	String with an \= equals sign = Cadena con un signo de \= igualdad
+```text
```
Comment (ShadowNinja) [inline README.md]: Something like `cfg` would probably yield better formatting.

File: README.md (hunk 1 above)
Comment (rdnuk) [inline README.md]: thanks didnt notice that one, il correct it now

**Comment by sofar** [comment]
0c7b304

**Pull request closed without merging**